datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -152,7 +152,7 @@
   private static final String EVENT_PRODUCER_CONFIG_DOMAIN = "brooklin.server.eventProducer";
 
   private static final String MODULE = Coordinator.class.getSimpleName();
-  private static final long EVENT_THREAD_JOIN_TIMEOUT = 1000L;
+  private static final Duration EVENT_THREAD_JOIN_TIMEOUT = Duration.ofSeconds(30);
```

Comment:
Since this is only used in a call to `Thread::join()`, I suggest (a) keeping it as a `long`, (b) adding a `_MILLIS` suffix, and (c) using `Duration` to initialize it, i.e.

```java
private static final long EVENT_THREAD_JOIN_TIMEOUT_MILLIS = Duration.ofSeconds(30).toMillis();
```